config/default/kustomization.yaml

```
@@ -0,0 +1,74 @@
+# Adds namespace to all resources.
+namespace: volume-replication-operator-system
```

```suggestion
namespace: volume-replication-operator
```

avoid using `system` suffix.

---

Created https://github.com/kube-storage/volume-replication-operator/issues/7